frontend/src/main/pages/MenuItemReview/MenuItemReviewIndexPage.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+import { useBackend } from 'main/utils/useBackend';
+import BasicLayout from "main/layouts/BasicLayout/BasicLayout";
+import MenuItemReviewsTable from 'main/components/MenuItemReviews/MenuItemReviewsTable';
```

Comment:
Do you have a file for this import? When I run on localhost I'm getting an error that this module doesn't exist and I can't find you MenuItemReviewsTable file. 
Currently failing test: 
40-check-production-build
![image](https://user-images.githubusercontent.com/59808746/169185051-a5ccfe54-d676-4d77-9e96-186c13f8b857.png)

Reply:
I included this file in the third pull request. Can you take a look at that? Sorry for that.